Magento2/ruleset.xml

```diff
@@ -242,6 +242,13 @@
         <type>warning</type>
     </rule>
 
+    <rule ref="Magento2.Legacy.EmailTemplate.FoundObsoleteEmailDirective">
+        <include-pattern>view/email/*.html</include-pattern>
+        <include-pattern>view/*/email/*.html</include-pattern>
+        <severity>8</severity>
+        <type>warning</type>
```

Comment:
I think this should be an error as using of obsolete directives will actually harm the functionality of email templates